Lexplorer/Pages/AccountDetails.razor

```diff
@@ -69,20 +69,31 @@
             }
             else
             {
+                
                 <MudGrid>
                     <MudItem xs="12" Class="d-flex justify-center">
-                        <MudToolBar>
-                            <MudIconButton Icon="@Icons.Filled.FirstPage" Disabled=@(gotoNFTPage == 1) OnClick="@(() => gotoNFTPage = 1)" />
-                            <MudIconButton Icon="@Icons.Filled.NavigateBefore" Disabled=@(gotoNFTPage == 1) OnClick="@(() => gotoNFTPage -= 1)" />
-                            <MudNumericField HideSpinButtons="true" @bind-Value="gotoNFTPage" Label="Page" Variant="Variant.Outlined" Min="1" Margin="Margin.Dense" Class="flex-grow-0" Style="width:50px;" />
-                            <MudIconButton Icon="@Icons.Filled.NavigateNext" Disabled=@(accountNFTSlots?.Count < nftPageSize) OnClick="@(() => gotoNFTPage += 1)" />
+                        <MudPaper Width="100%" >
+                            
+                        <MudToolBar Class="justify-center">
+                            <MudPagination Size="Size.Small" ShowFirstButton="true" 
+                                BoundaryCount="1" 
+                                MiddleCount="1" 
+                                ShowLastButton="true" 
+                                Selected="@currentPage"
+                                NextIcon="@Icons.Filled.NavigateBefore" 
+                                FirstIcon="@Icons.Filled.LastPage"
+                                BeforeIcon="@Icons.Filled.NavigateNext"
+                                LastIcon="@Icons.Filled.FirstPage"
```

Comment:
These icon names look mixed up - and they actually do show up mixed on screen for me too - but not in your screenshot?

Reply:
Thats messed up.. for me, when I use the default pager it shows wrong icons. I thought it was a bug in mudblazor so thats why I changed it. If its actually wrong I'll change it to default. I have to test with more browsers etc. Give it a go you too on multiple browsers. It probably local at my environment.

Reply:
I've tried removing that part where I set the icons, and this is how it looks now:
![image](https://user-images.githubusercontent.com/58626043/174249631-b3682735-0e47-4048-9e2e-8a18abb332e7.png)
Is this correct? Cause it looks weird to me

Reply:
No this is not correct - but strangely enough, this is how it always looked for me, without you changing it! I'd say it looks like something is messed up locally on your side, but of course I'm just guessing. Do you have a chance to test on a different machine? Without deploying to dev, I can't.

Reply:
Now with your latest changes, the icons are correct for the first time for me:

<img width="209" alt="image" src="https://user-images.githubusercontent.com/44807458/174269333-32ff240c-360a-4f72-96c3-b77f9ea8e397.png">

Reply:
It must be something with my environment... I'll leave it as it is now :) Just have to make the same change on the hidden one

Reply:
OK, both look fine now for me, with Safari on various resolutions, including mobile.